packages/orbit-components/src/Box/README.md

```diff
@@ -0,0 +1,161 @@
+| position     | [`position`](#position)                                              |         | The `position` of the Box        |
+| left         | `string`                                                             |         | The `left` of the Box            |
+| top          | `string`                                                             |         | The `top` of the Box             |
+| right        | `string`                                                             |         | The `right` of the Box           |
+| bottom       | `string`                                                             |         | The `bottom` of the Box          |
+| textAlign    | [`textAlign`](#textAlign)                                            |         | The `text-align` of the Box      |
+| elevation    | [`elevation`](#elevation)                                            |         | The `box-shadow` of the Box      |
+| color        | `string`                                                             |         | The `color` of the Box           |
+| background   | `string`                                                             |         | The `background` of the Box      |
+| borderRadius | [`borderRadius`](#borderRadius)                                      |         | The `border-radius` of the Box   |
+| overflow     | [`overflow`](#overflow)                                              |         | The `overflow` of the Box        |
+
+### enum
+
+| justify     | direction          | align       | wrap       | shrink | width    | height   |
+| :---------- | :----------------- | :---------- | :--------- | :----- | :------- | :------- |
+| `"start"`   | `"row"`            | `"start"`   | `"nowrap"` | `0`    | `"full"` | `"full"` |
+| `"end"`     | `"column"`         | `"end"`     | `"wrap"`   | `1`    | `"auto"` | `"auto"` |
+| `"center"`  | `"row-reverse"`    | `"center"`  |            |        |          |          |
+| `"between"` | `"column-reverse"` | `"stretch"` |            |        |          |          |
+| `"around"`  |                    |             |            |        |          |          |
+
+### position
+
+| position     |
+| :----------- |
+| `"relative"` |
+| `"absolute"` |
+| `"fixed"`    |
+
+### textAlign
+
+| textAlign  |
+| :--------- |
+| `"left"`   |
+| `"center"` |
+| `"right"`  |
+
+### borderRadius
+
+| borderRadius     |
+| :--------------- |
+| `"small"`        |
+| `"normal"`       |
+| `"large"`        |
+| `"radiusCircle"` |
+
+### overflow
+
+| overflow    |
+| :---------- |
+| `"auto"`    |
+| `"hidden"`  |
+| `"scroll"`  |
+| `"visible"` |
+
+### elevation
+
+| elevation   |
+| :---------- |
+| `"action"`  |
+| `"fixed"`   |
+| `"overlay"` |
+| `"raised"`  |
+
+### spacingObject
+
+| key        | value                          |
+| :--------- | :----------------------------- |
+| `"top"`    | [`spacingToken](#spacingtoken) |
```

Comment:
Missing backtick on all the lines. Also it was not clear to me, how I should define the `spacingObject`, because it refers to it very oddly.

Would attach simple example or note below the `spacingObject` title, e.g.:
```
If you want to define `spacingObject`, you can define specific directions like this:
<Box margin={{ top: "XXSmall", bottom: "XXSmall" }} />
```